examples/base/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@raincatcher/base",
+  "version": "1.0.0",
+  "description": "NPM package template for the RainCatcher project",
+  "main": "index.js",
+  "author": "feedhenry-raincatcher@redhat.com",
+  "license": "Apache-2.0",
+  "scripts": {
+    "build": "del lib && tsc",
+    "test": "nyc mocha",
+    "prepublish": "npm run build"
+  },
+  "nyc": {
+    "include": [
+      "src/**/*.ts"
+    ],
+    "extension": [
+      ".ts"
+    ],
+    "require": [
+      "ts-node/register"
+    ],
+    "check-coverage": true,
+    "lines": 75,
+    "functions": 100,
+    "branches": 80
+  },
+  "devDependencies": {
+    "@types/mocha": "^2.2.41",
+    "del-cli": "^1.0.0",
+    "mocha": "^3.4.2",
+    "nyc": "^11.0.1",
+    "source-map-support": "^0.4.15",
```

Comment:
@paolobueno - Point 2 - seems like standard aproach to me, but I will need to dig little bit deeper into this to make sure that this is the right aproach.

[IDEA] - Can we have unit test setup as separate module that will include best patterns etc. 
or we will relay on configuration from base module. Just thinking about the best update scenario for modules etc.